### PR TITLE
Inconsistencies Part 1 (examples/)

### LINKED ISSUE #245: Address inconsistencies in comments, naming, casing, etc.. Part 1 for the `examples` directory
Throughout the code base there are inconsistencies in various aspects of the comments, naming, casing, etc.

As a developer/user, I want consistent comments, naming, casing, etc. so that when I read the code, or see output it is consistent from one interaction to the next.

In the `examples` folder:
- [x] sometimes the output string reads "...with the given name..." sometimes it reads "...named xyz...".
- [x] sometimes the `It` block is capitalized, sometimes it is not (it should be capitalized).
- [x] `.md` files sometimes the "replace" variables are aligned, sometimes they're not.
- [x] Using conjugated names instead of full names in the description string of unit tests and examples (e.g., `keyVault` instead of `Key Vault`, `EventHub` instead of `Event Hub`, `Sql` or `sql` instead of `SQL`; i.e., the proper names).
- [x] splatted variables are aligned, sometimes they're not.
- [x] Locally defined variables should be lower cased
- [x] Update file encoding to UTF-8 w/BOM
- [x] import and export Az.InfrastructureTesting
- [x] Add `PropertyType` and `PropertyKey` to descriptions in `Confirm-AzBPResource` examples

Acceptance criteria:
- [ ] Thoroughly go through the code base to find any inconsistencies
- [ ] Correct the inconsistencies

### PR DESCRIPTION
**Pull request by msha1026**
Closes #245.

### CODE REVIEW

File: examples/CosmosDB/CosmosDB.Tests.ps1 [hunk 1]
```diff
@@ -149,7 +149,7 @@ Describe 'Verify Cosmos DB Account' {
 
 #######################################################################################################################
 
-  It "Should contain a sql Cosmos DB Account named $sqlAccountName - Confirm-AzBPResource" {
+  It "Should contain a Sql Cosmos DB Account named $sqlAccountName - Confirm-AzBPResource" {
```
Comment (marclerwick) [inline examples/CosmosDB/CosmosDB.Tests.ps1:152]: Fully capitalize `SQL`

File: examples/SqlDatabase/SqlDatabase.Tests.ps1 [hunk 2]
```diff
@@ -12,7 +12,7 @@ Describe 'Verify Sql Database' {
     $Script:noDatabaseName = 'nosamplesqlserver'
   }
 
-  It 'Should contain a sql database with given name - Confirm-AzBPResource' {
+  It "Should contain a Sql Database named $databaseName - Confirm-AzBPResource" {
```
Comment (marclerwick) [inline examples/SqlDatabase/SqlDatabase.Tests.ps1:15]: Fully capitalize `SQL`

**Review by marclerwick — CHANGES_REQUESTED** [review]

File: examples/ResourceGroup/ResourceGroup_Example.md [hunk 3]
```diff
@@ -23,7 +23,7 @@
 
 1. Update `ResourceGroup.Tests.ps1` variables to point to your expected resources:
 
-   - `rgtest` -> `your-resource-group-name`
+   - `rgtest`    -> `your-resource-group-name`
    - `notRgName` -> `not-your-resource-group-name`
```
Comment (ozboms) [inline examples/ResourceGroup/ResourceGroup_Example.md:28]: Could you add a location parameter here?

```suggestion
`westus3` -> `your-resource-group-location`
```

File: examples/SqlDatabase/SqlDatabase.Tests.ps1 [hunk 4]
```diff
@@ -37,21 +37,24 @@ Describe 'Verify Sql Database' {
...
+      DatabaseName      = $databaseName
+      ServerName        = $serverName
     }
 
+    # act and assert
     Confirm-AzBPSqlDatabase @params | Should -BeSuccessful
   }
 
-  It 'Should not contain a Sql Database with the given name' {
+  It 'Should not contain a SQL Database with the given name' {
```
Comment (ozboms) [inline examples/SqlDatabase/SqlDatabase.Tests.ps1:56]:
```suggestion
  It 'Should not contain a SQL Database named $noDatabaseName' {
```

File: examples/SqlServer/SqlServer.Tests.ps1 [hunk 5]
```diff
@@ -34,27 +34,27 @@ Describe 'Verify Sql Server' {
...
     Confirm-AzBPResource @params | Should -BeSuccessful
   }
 
-  It "Should contain a Sql Server named $serverName" {
+  It "Should contain a SQL Server named $serverName" {
     Confirm-AzBPSqlServer -ResourceGroupName $rgName -ServerName $serverName | Should -BeSuccessful
   }
 
-  It 'Should not contain a Sql Server with the given name' {
+  It 'Should not contain a SQL Server with the given name' {
```
Comment (ozboms) [inline examples/SqlServer/SqlServer.Tests.ps1:45]:
```suggestion
  It 'Should not contain a SQL Server named $noServerName' {
```

**Review by quigg-caroline — APPROVED** [review]

**Review by marclerwick — APPROVED** [review]

**Pull request merged**